.github/workflows/pr_tests.yml

```diff
@@ -17,6 +17,7 @@ env:
   OMP_NUM_THREADS: 4
   MKL_NUM_THREADS: 4
   PYTEST_TIMEOUT: 60
+  HUGGINGFACE_CO_STAGING: False
```

Comment:
@Wauplin I implemented `is_staging_test` following: https://github.com/huggingface/transformers/blob/6247d1b2b6a3aea7c1484d9c6560a5795d35b7e0/tests/test_modeling_utils.py#L1041

But even when a test class is not decorated with it, the CI seems to pick up the staging env.

Reply:
Try maybe `HUGGINGFACE_CO_STAGING: 0`. I think that the environment variable is set to the string value `"False"` which evaluates to the boolean value `True`. That's my hypothesis at least :grimacing:

Reply:
Sorry for not being clear. 

I meant when I set it to True, the test cases not decorated with `is_staging_test` fail. Hence I set it to False.

Reply:
> I meant when I set it to True, the test cases not decorated with is_staging_test fail.

Yes, this is expected. If you set `HUGGINGFACE_CO_STAGING: 1` it tells `huggingface_hub` to always use the staging endpoint by default which make the tests fail. Either you should use this env variable which affects all tests, either you should use `is_staging_test` decorator to precisely describe which tests require the staging endpoint. I would go with solution 2 (i.e. not setting `HUGGINGFACE_CO_STAGING` at all and use `is_staging_test` when necessary).